script/bigscreenplayer.js

```diff
@@ -132,11 +132,10 @@ define('bigscreenplayer/bigscreenplayer',
 
         subtitles = Subtitles(
           playerComponent,
-          bigscreenPlayerData.media.captions || { captionsUrl: bigscreenPlayerData.media.captionsUrl },
```

Comment:
Can't you maintain backwards compatibility with the old API? This shouldn't be too hard to do

Reply:
We could, I'm not sure we need to as media selector will always be returning a captions object now